cfme/fixtures/bzs.py

```diff
@@ -23,16 +23,12 @@ def pytest_addoption(parser):
 def pytest_collection_modifyitems(session, config, items):
     if not config.getvalue("generate_bz_report"):
         return
-    store.terminalreporter.write("Loading automated/covered BZs ...\n", bold=True)
+    store.terminalreporter.write("Loading automated BZs ...\n", bold=True)
     bz_list = []
     for item in items:
-        if "automates" not in item._metadata and "coverage" not in item._metadata:
-            continue
```

Comment:
@prichard77 rather than removing the handling of the `coverage`/manual cases altogether, I think we should make it an option (`--include-manual`) that defaults to `False`. 

Something like
```   
group.addoption(
        '--include-manual',
        action='store_true',
        default=False,
        dest='include_manual',
        help='Include manual coverage in the report'
    )
```

Reply:
Then the `miq bz` command can also have an `--include-manual` flag

Reply:
Sounds like a great idea. working on it...

Reply:
OH that's right, we should just use that! So we can just add an `--include-manual` flag in `miq bz coverage` and then include it in the `pytest.main` call based on that.

Reply:
That way no changes are required to `cfme.fixtures.bzs.py`